src/js/Util.js

```diff
@@ -0,0 +1,24 @@
+var Util = {
```

Comment:
Why is this file include in the plugin?
Not needed. It's called indrectly if you call for example:
`
PluginEvents.add("MY_CONSTANT");
`